package.json

```diff
@@ -18,13 +19,30 @@
     "string-to-color": "^2.0.0"
   },
   "devDependencies": {
+    "babel-eslint": "^8.2.5",
     "babel-jest": "23.0.1",
     "babel-preset-react-native": "4.0.0",
+    "husky": "^0.14.3",
     "jest": "23.1.0",
+    "lint-staged": "^7.2.0",
     "react-test-renderer": "16.3.1",
     "standard": "^11.0.1"
   },
   "jest": {
     "preset": "react-native"
+  },
+  "lint-staged": {
+    "*.js": [
+      "standard --fix",
+      "git add"
+    ]
+  },
+  "standard": {
+    "parser": "babel-eslint",
+    "globals": [
+      "alert",
```

Comment:
`process` also? Weird that it didn't catch it, or it's already included.

Reply:
guess it's included